dbt_project.yml

```diff
@@ -5,7 +5,7 @@ require-dbt-version: [">=1.3.0", "<2.0.0"]
 
 models:
   qualtrics_source:
-    # +schema: stg_qualtrics
+    # +schema: stg_qualtrics 
```

Comment:
Is there a reason this is commented out? Additionally, can we adjust this to be in line with the new approach of having the custom schema be `qualtrics_source` to match the name of the package.

Reply:
ah yes will change to `qualtrics_source`

and i had it commented out since i was _trying_ to run this on the customer's dataset and didn't want to create multiple schemas

Reply:
@fivetran-jamie Sounds good!

Also, you don't need to edit this file to not create multiple schemas. We can use the same steps outlined in [Changing the Build Schema](https://github.com/fivetran/dbt_qualtrics_source#changing-the-build-schema) section to override the custom schema. You just add the following to ensure the models are written in a single schema.

```yml
## root dbt_project.yml

models:
   +schema:
```

Reply:
![image](https://github.com/fivetran/dbt_qualtrics_source/assets/65564846/547e0a44-b763-412e-a5b4-7228b48d5ea0)